src/components/navigation.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const links = [{ path: '/', text: 'Motorbick' },
+  { path: 'newpath', text: 'Reserve' },
+  { path: 'newpath2', text: 'My reservations' },
+  { path: 'newpath3', text: 'ADD Motorbick' },
+  { path: 'newpath3', text: 'Delete Motorbick' },
```

Comment:
Hi, @AliShahSafdari 😃
Well done on creating the side navigation bar ✔👏

Reply:
Thanks